tests/README.md

```diff
@@ -47,28 +47,33 @@ Use [helm.sh](/tests/scripts/helm.sh) to install helm and set up rook charts def
 **Note:**  *kubeadm.sh, minikube.sh and helm.sh scripts depend on some artifacts under _output dir generated during build time, these scripts 
 should be run from project root. eg* `tests/script/kubeadm.sh up`
 
-### Run Tests
-From the root do the following:
-1. Build rook: `make build`
-2. Start kubernetes using one of the following:
+## Run Tests
+From the project root do the following:
+#### 1. Build rook: 
+ run `make build`
+
+#### 2. Start kubernetes 
+using one of the following:
    
    -  using Kubeadm  
    ```
     tests/scripts/kubeadm.sh up
-    tests/scripts/helm.sh up
-    ```
+    tests/scripts/helm.sh up  
+  ```   
    - using minikube
    ```
    tests/scripts/minikube.sh up
-   tests/scripts/minikube.sh update
    tests/scripts/minikube.sh helm
    tests/scripts/helm.sh up
    ```
-3. Run integration tests: `make test-integration`
 
+#### 3. Run integration tests: 
+Integration tests can be run using tests binary `_output/tests/${platform}/integration` that is generated during build time
```

Comment:
isn't `make test-integration` the recommended way to run the tests? I'm still confused about the multiple options that we are trying to support in this doc